easybuild/easyblocks/c/cplex.py

```diff
@@ -110,11 +110,21 @@ def post_install_step(self):
             raise EasyBuildError("No bins found using %s in %s", binglob, self.installdir)
 
     def make_module_extra(self):
-        """Add installdir to path and set CPLEX_HOME"""
+        """Add bin dirs and lib dirs and set CPLEX_HOME and CPLEXDIR"""
+
+        os.chdir(self.installdir)
+        binglob = '*/bin/x86-64*'
+        bins = glob.glob(binglob)
+        libglob = '*/lib/x86-64*/*pic'
+        libs = glob.glob(libglob)
 
         txt = super(EB_CPLEX, self).make_module_extra()
-        txt += self.module_generator.prepend_paths("PATH", [self.bindir])
+
+        txt += self.module_generator.prepend_paths("PATH", [path for path in bins])
+        txt += self.module_generator.prepend_paths("LD_LIBRARY_PATH", reduce(lambda x, y: x+y, [bins,libs]))
+
         txt += self.module_generator.set_environment("CPLEX_HOME", os.path.join(self.installdir, 'cplex'))
+        txt += self.module_generator.set_environment("CPLEXDIR", os.path.join(self.installdir, 'cplex'))
```

Comment:
hmm, do we have a reference for these? I'm wondering why one is using `_`, and another is not...